script.colorbox/resources/lib/Utils.py

```diff
@@ -94,37 +109,6 @@ def Shuffle_Set(amount,timed=40):
         npegs = ''.join(npeg)
         HOME.clearProperty('Colorbox_shuffle.' + npegs)
         time.sleep(timed)
```

Comment:
Can you please use xbmc.sleep here instead of python's time? If the time for the sleep is bigger than 5 seconds please use `xbmc.Monitor().waitforabort(timeinseconds)`